src/pcapi/core/offers/api.py

```diff
@@ -709,3 +726,16 @@ def import_offer_validation_config(config_as_yaml: str, user: User = None) -> Of
     config = OfferValidationConfig(specs=config_as_dict, user=user)
     repository.save(config)
     return config
+
+
+def _load_product_by_isbn_and_check_is_gcu_compatible_or_raise_error(isbn: str) -> Product:
+    product = Product.query.filter(Product.extraData["isbn"].astext == isbn).first()
+    if product is None or product.isGcuCompatible is False:
```

Comment:
(style)
```suggestion
    if product is None or not product.isGcuCompatible:
```

(comparer avec `is False` fonctionne, mais ça n'est pas courant en Python)